module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -185,15 +185,17 @@ public function getSortTitle()
      * @param int    $ownerId ID of user to check for ownership
      *
      * @return array
+     *
+     * @deprecated Use TagServiceInterface::getTagsForRecord() or TagServiceInterface::getListTagsForRecord()
+     * or TagServiceInterface::getNonListTagsForRecord()
      */
     public function getTags(
         $list_id = null,
         $user_id = null,
         $sort = 'count',
         $ownerId = null
     ) {
-        $tags = $this->getDbService(TagServiceInterface::class);
-        return $tags->getForResource(
+        return $this->getDbTable('Tags')->getForResource(
```

Comment:
I reverted this to use the legacy logic since the service logic is no longer equivalent.